mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-plugins = strawberry.ext.mypy_plugin
+plugins = pydantic.mypy, strawberry.ext.mypy_plugin
 implicit_reexport = False
```

Comment:
unfortunately we need the pydantic plugin to run the mypy test for to_pydantic. because the mypy logic to resolve what fields are present depends on pydantic